assets/i18next.js

```diff
@@ -0,0 +1,37 @@
+i18next
+  .use(i18nextXHRBackend)
+  .use(i18nextBrowserLanguageDetector)
+  .init({
+    fallbackLng: 'zh',
+    whitelist: ['en', 'zh'],
+    debug: true,
+    backend: {
+      loadPath: 'https://gist.githubusercontent.com/patcon/1f68d3efd30360048bce81b3a7655c5b/raw/{{lng}}.json',
```

Comment:
this should def point somewhere else, and prob to a raw file on github repo, or a json file hosted as part of app

Reply:
yes I think it's better to also include these two json files in the PR